lib/session.js

```diff
@@ -150,8 +158,8 @@ SessionStore.prototype.createSession = function(sid, fn) {
 /**
 * Get the already created session
 */
-SessionStore.prototype.getSession = function (uid) {
-  return userSessionIndex[uid];
+SessionStore.prototype.getSession = function (uid, sid) {
```

Comment:
Previous implementation was broken, a single uid can have multiple sessions. New signature is more correct.